node/out_request.js

```diff
@@ -35,9 +36,14 @@ function TChannelOutRequest(id, options) {
     self.responseEvent = self.defineEvent('response');
     self.finishEvent = self.defineEvent('finish');
 
+    assert(options.channel, 'channel required');
+
     self.logger = options.logger;
     self.random = options.random;
     self.timers = options.timers;
+    self.retryCount = options.retryCount;
```

Comment:
Why you need to pass in the retryCount? Shouldn't it be set to 0 when out request is initialized?